src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3196,31 +3203,31 @@ void visitArgumentEvaluateAndUnsplit(int argumentIndex, bool assertsTrue, bool a
                     break;
                 case RefKind.Ref:
                     {
-                        if (!argument.IsSuppressed)
+                        if (!argumentNoConversion.IsSuppressed)
                         {
                             var lvalueResultType = result.LValueType;
                             if (IsNullabilityMismatch(lvalueResultType.Type, parameterType.Type))
                             {
-                                // declared types must match, ignoring top-level nullability
```

Comment:
>, ignoring top-level nullability [](start = 60, length = 32)

It looks like this comment still applies.

Reply:
Whoops, I think this was a merge mixup.

---
In reply to: [281675740](https://github.com/dotnet/roslyn/pull/35544#discussion_r281675740) [](ancestors = 281675740)